data/hooks/conf_regen/15-nginx

```diff
@@ -38,6 +38,15 @@ do_pre_regen() {
   for domain in $domain_list; do
     domain_conf_dir="${nginx_conf_dir}/${domain}.d"
     mkdir -p "$domain_conf_dir"
+    domain_ca_type=$(yunohost domain cert-status ${domain} | grep CA_type)
+    if [$domain_ca_type = "CA_type: Let's Encrypt"]; then
```

Comment:
typo, there should be a space after the "[" otherwise you'll get a syntax error in bash.

Have you tested this code? Same for the other "]".

Also the variable should be quoted here because of spaces.

Reply:
Well indeed I change it in the HPKP PR :) I will correct that.